### PR TITLE
Update net zero pledges notebook with 2023 data

### PR DESCRIPTION
**Pull request by dgounaris**
Updates net zero pledges notebook with 2023 data.

### CODE REVIEW

**Review by mgrabovsky — APPROVED** [review]
Looks great. Thanks for the update!

Just one small thing: Could you please delete the outdated Net Zero Tracker snapshot, `data/net-zero-tracker/snapshot_2022-03-30_05-05-02.xlsx`, as we aren't using it anymore?

File: notebooks/net-zero-pledges.ipynb [hunk 1]
```diff
@@ -51,26 +51,26 @@
    "source": [
     "## CO2 emissions data\n",
     "\n",
-    "Emissions data are loaded from EDGAR database (Emissions Database for Global Atmospheric Research) published by Joint Research Centre of European Commission. We use the newest version [EDGAR v6.0](https://edgar.jrc.ec.europa.eu/dataset_ghg60) that contains emissions data for CO2, CH4 and N2O for all countries up to 2018. The values are given in Gg (gigagrams, i.e. thousands of tonnes).\n",
+    "Emissions data are loaded from EDGAR database (Emissions Database for Global Atmospheric Research) published by Joint Research Centre of European Commission. We use the newest version [EDGAR v8.0](https://edgar.jrc.ec.europa.eu/dataset_ghg80) that contains emissions data for CO2, CH4 and N2O for all countries up to 2018. The values are given in Gg (gigagrams, i.e. thousands of tonnes).\n",
```
Comment (mgrabovsky) [inline notebooks/net-zero-pledges.ipynb:54]:
```suggestion
    "Emissions data are loaded from EDGAR database (Emissions Database for Global Atmospheric Research) published by Joint Research Centre of European Commission. We use the newest version [EDGAR v8.0](https://edgar.jrc.ec.europa.eu/dataset_ghg80) that contains emissions data for CO2, CH4 and N2O for all countries up to 2020. The values are given in Gg (gigagrams, i.e. thousands of tonnes).\n",
```

**Comment by dgounaris** [comment]
> Looks great. Thanks for the update!
> 
> Just one small thing: Could you please delete the outdated Net Zero Tracker snapshot, `data/net-zero-tracker/snapshot_2022-03-30_05-05-02.xlsx`, as we aren't using it anymore?

Thought we want them for historical reasons, similar to v5 and v6 of the EDGAR datasets (?).

Thanks a lot for the prompt review! I'll address the SVG thing soon and get back to you.

**Comment by mgrabovsky** [comment]
> Thought we want them for historical reasons, similar to v5 and v6 of the EDGAR datasets (?).

I'd prefer to remove unused datasets. We can always retrieve them from Git history or from the original source should the need arise. The EDGAR v6.0 dataset is actually still used in the `actors-in-climate-negotiations` notebook (which we don't plan on updating at the moment), but EDGAR v5.0 is not used either.

**Comment by dgounaris** [comment]
@mgrabovsky  Updated, let me know if it looks good.

**Review by mgrabovsky — APPROVED** [review]
Looks great! Thanks again.

**Pull request merged**